src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -139,7 +139,7 @@ IConfiguration configuration
                 }
 
                 // Return review URL created for current package if exists
-                var review = prReviews.SingleOrDefault(r => r.PackageName == packageName);
+                var review = prReviews.SingleOrDefault(r => r.PackageName == packageName && r.Language == language);

```

Comment:
This will fail for existing PR records and regular language PRs where no language is set. We always needs to consider backward compatibility too.

Reply:
out of curiosity, in which cases is language not set?

Reply:
I think when `r.language` is not set, `language` should also be `null`, then the condition should be meet.

Reply:
> out of curiosity, in which cases is language not set?

@maririos : Language is a new property we added for SDK automation. So all existing pull requests from regular language does not have this property in cosmos DB. It will be only set for SDK automation now. I will submit another PR in individual language pipelines to set this language and it's an optional parameter. My suggestion was to keep it back-word compatible.